TestAdmin_BlockedList and rest api Flaky tests

Sometimes `docker-compose -f .\compose-dev.yml build` fails with following error:
```
2018/07/01 02:47:26 [INFO] REST PUT - /api/v1/admin/user/user2?site=radio-t&block=1&ttl=10ms - 127.0.0.1 - 200 (53) - 15.3795ms  - dev "developer one"

2018/07/01 02:47:26 [INFO] REST PUT - /api/v1/admin/user/user2?site=radio-t&block=1&ttl=10ms - dbc7c999343f - 200 (53) - 15.4186ms  - dev "developer one"
2018/07/01 02:47:26 [INFO] REST GET - /api/v1/admin/blocked?site=radio-t - 127.0.0.1 - 200 (70) - 97.1µs  - dev "developer one"
2018/07/01 02:47:26 [INFO] REST GET - /api/v1/admin/blocked?site=radio-t - dbc7c999343f - 200 (70) - 127.3µs  - dev "developer one"
--- FAIL: TestAdmin_BlockedList (0.05s)
        Error Trace:    admin_test.go:258
        Error:          Not equal:
                        expected: 2
                        actual  : 1
        Test:           TestAdmin_BlockedList
        Messages:       two users blocked
panic: runtime error: index out of range [recovered]
        panic: runtime error: index out of range

goroutine 103 [running]:
testing.tRunner.func1(0xc4202400f0)
        /usr/local/go/src/testing/testing.go:742 +0x29d
panic(0x920f20, 0xcb8950)
        /usr/local/go/src/runtime/panic.go:502 +0x229
github.com/umputun/remark/backend/app/rest/api.TestAdmin_BlockedList(0xc4202400f0)
        /go/src/github.com/umputun/remark/backend/app/rest/api/admin_test.go:260 +0xc1e
testing.tRunner(0xc4202400f0, 0x9ddad8)
        /usr/local/go/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:824 +0x2e0
FAIL    github.com/umputun/remark/backend/app/rest/api  5.886s
ok      github.com/umputun/remark/backend/app/rest/auth 0.447s
```
Have to rerun multiple times to get this tests working. After then test layer is cached so you don't see test errors anymore until new changes in sources.

more logs:
```
--- FAIL: TestBoltAdmin_BlockUserWithTTL (0.08s)
        Error Trace:    bolt_admin_test.go:149
        Error:          Should be true
        Test:           TestBoltAdmin_BlockUserWithTTL
        Messages:       user1 blocked
```
```
2018/07/01 03:04:47 [INFO] REST GET - /api/v1/admin/blocked?site=radio-t - dbc7c999343f - 200 (70) - 201.7µs  - dev "developer one"
--- FAIL: TestAdmin_BlockedList (0.06s)
        Error Trace:    admin_test.go:258
        Error:          Not equal:
                        expected: 2
                        actual  : 1
        Test:           TestAdmin_BlockedList
        Messages:       two users blocked
panic: runtime error: index out of range [recovered]
        panic: runtime error: index out of range

goroutine 78 [running]:
testing.tRunner.func1(0xc4202460f0)
        /usr/local/go/src/testing/testing.go:742 +0x29d
panic(0x920f20, 0xcb8950)
        /usr/local/go/src/runtime/panic.go:502 +0x229
github.com/umputun/remark/backend/app/rest/api.TestAdmin_BlockedList(0xc4202460f0)
        /go/src/github.com/umputun/remark/backend/app/rest/api/admin_test.go:260 +0xc1e
testing.tRunner(0xc4202460f0, 0x9ddad8)
        /usr/local/go/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:824 +0x2e0
FAIL    github.com/umputun/remark/backend/app/rest/api  5.885s
```
Looks like smth wrong with blocking users with ttl.
For some reason it didn't blocks immediately. Adding `time.Sleep(1 * time.Millisecond)` before corresponding asserts in admin_test.go:253, bolt_admin_test.go::149 and bolt_admin_test.go::163 helps to pass all tests. But I guess it isn't solution we expect.

--- a/backend/app/rest/api/admin_test.go
+++ b/backend/app/rest/api/admin_test.go
@@ -244,7 +244,7 @@ func TestAdmin_BlockedList(t *testing.T) {
 
 	// block user2
 	req, err = http.NewRequest(http.MethodPut,
-		fmt.Sprintf("%s/api/v1/admin/user/%s?site=radio-t&block=%d&ttl=10ms", ts.URL, "user2", 1), nil)
+		fmt.Sprintf("%s/api/v1/admin/user/%s?site=radio-t&block=%d&ttl=50ms", ts.URL, "user2", 1), nil)
 	assert.Nil(t, err)
 	req.SetBasicAuth("dev", "password")
 	_, err = client.Do(req)
@@ -259,7 +259,7 @@ func TestAdmin_BlockedList(t *testing.T) {
 	assert.Equal(t, "user1", users[0].ID)
 	assert.Equal(t, "user2", users[1].ID)
 
-	time.Sleep(11 * time.Millisecond)
+	time.Sleep(50 * time.Millisecond)
 	res, code = getWithAuth(t, ts.URL+"/api/v1/admin/blocked?site=radio-t")
 	require.Equal(t, 200, code, res)
 	users = []store.BlockedUser{}

--- a/backend/app/store/engine/bolt_admin_test.go
+++ b/backend/app/store/engine/bolt_admin_test.go
@@ -145,9 +145,9 @@ func TestBoltAdmin_BlockUserWithTTL(t *testing.T) {
 	defer os.Remove(testDb)
 	b := prep(t)
 	assert.False(t, b.IsBlocked("radio-t", "user1"), "nothing blocked")
-	assert.NoError(t, b.SetBlock("radio-t", "user1", true, 10*time.Millisecond))
+	assert.NoError(t, b.SetBlock("radio-t", "user1", true, 50*time.Millisecond))
 	assert.True(t, b.IsBlocked("radio-t", "user1"), "user1 blocked")
-	time.Sleep(11 * time.Millisecond)
+	time.Sleep(50 * time.Millisecond)
 	assert.False(t, b.IsBlocked("radio-t", "user1"), "user1 un-blocked automatically")
 }
 
@@ -156,7 +156,7 @@ func TestBoltAdmin_BlockList(t *testing.T) {
 	b := prep(t)
 
 	assert.NoError(t, b.SetBlock("radio-t", "user1", true, 0))
-	assert.NoError(t, b.SetBlock("radio-t", "user2", true, 10*time.Millisecond))
+	assert.NoError(t, b.SetBlock("radio-t", "user2", true, 50*time.Millisecond))
 	assert.NoError(t, b.SetBlock("radio-t", "user3", false, 0))
 
 	ids, err := b.Blocked("radio-t")
@@ -167,7 +167,7 @@ func TestBoltAdmin_BlockList(t *testing.T) {
 	assert.Equal(t, "user2", ids[1].ID)
 	t.Logf("%+v", ids)
 
-	time.Sleep(11 * time.Millisecond)
+	time.Sleep(50 * time.Millisecond)
 	ids, err = b.Blocked("radio-t")
 	assert.NoError(t, err)
 	assert.Equal(t, 1, len(ids))

